### PR TITLE
feat(web,server): server features

### PR DESCRIPTION
**Pull request by jrasm91**
Add a new endpoint `/server-info/features`, which returns what is enabled on the server, in order to allow for some dynamic web rendering when machine learning is disabled, etc.

- Hide "Explore" tab when search is disabled
- Hide search bar when search is disabled
- Disable clip, object tagging, and facial recognition jobs when machine learning is disabled 

Use via `$featureFlags.machineLearning`, etc.

In the future I would like to use this to control "manual asset jobs" and get the login auth/oauth/password configuration from this endpoint instead of `oauth/config`.

image

### CODE REVIEW

**Comment by alextran1502** [comment]
If the job is disabled, should we not show it?

**Comment by jrasm91** [comment]
I thought about doing that originally. I'm not sure what is better to hide it or disable it. If it is hidden it might cause confusion if there is no indication what or why it is hidden. If the docs mention a job and you can't find it, it could be quite frustrating.

File: web/src/lib/components/admin-page/jobs/job-tile.svelte [hunk 1]
```diff
@@ -94,7 +96,15 @@
...
   </div>
   <div class="flex w-full flex-row overflow-hidden sm:w-32 sm:flex-col">
-    {#if !isIdle}
+    {#if disabled}
+      <JobTileButton
+        disabled={true}
+        color="light-gray"
+        on:click={() => dispatch('command', { command: JobCommand.Start, force: false })}
+      >
+        <AlertCircle size="48" /> DISABLED
```
Comment (alextran1502) [inline web/src/lib/components/admin-page/jobs/job-tile.svelte:105]: Can we make the size a bit smaller? 36 is probably good

**Review by alextran1502 — APPROVED** [review]

File: web/src/lib/components/admin-page/jobs/job-tile.svelte (hunk 1 above)
Comment (jrasm91) [reply web/src/lib/components/admin-page/jobs/job-tile.svelte:105]: Sure, I just copied it from the play icon.

**Pull request merged**